src/util.ts

```diff
@@ -347,12 +347,12 @@ export async function writeOutput(
   const outputToSimpleString = (output: EvaluateTableOutput) => {
     const passFailText = output.pass ? '[PASS]' : '[FAIL]';
     const namedScoresText = Object.entries(output.namedScores)
-      .map(([name, value]) => `${name}: ${value.toFixed(2)}`)
+      .map(([name, value]) => `${name}: ${value?.toFixed(2)}`)
       .join(', ');
     const scoreText =
       namedScoresText.length > 0
-        ? `(${output.score.toFixed(2)}, ${namedScoresText})`
-        : `(${output.score.toFixed(2)})`;
+        ? `(${output.score?.toFixed(2)}, ${namedScoresText})`
```

Comment:
this is the cause of your error. Your derived metric did not return a score! Thank you for making the UI better.